src/actions/guides/database/migrations.cr

```diff
@@ -0,0 +1,231 @@
+class Guides::Database::Migrations < GuideAction
+  guide_route "/database/migrations"
+
+  def self.title
+    "Migrations"
+  end
+
+  def markdown
+    <<-MD
+    ## Migrating Data
+
+    ### Introduction
+
+    You can think of migrations like version control for your database. These are classes that are scoped with a timestamp to allow you to update your database in versions as well as undo changes that you've made previously while keeping track of the order you made them in.
+
+    ### Generating a new one
+
+    To create a new migration, run the `lucky gen.migration {PurposeOfMigration}`. (e.g. If you need to add a `name` column to `users`, you would run `lucky gen.migration AddNameToUsers`). This would generate a `db/migrations/#{Time.utc.to_s("%Y%m%d%H%I%S")}_add_name_to_users.cr` file.
+
+    > When you generate a new model, a migration file is created for you.
+
+    ### Anatomy
+
+    A new migration file will always start with a timestamp. This is to keep the order in which they are ran. You may have a migration that adds a column, and then a year later, another one that removes that column. You want to run these in the correct order.
```

Comment:
Maybe file -> file name to make it clearer. Love the explanation of why this is important!